testing/src/test/scala/org/http4s/Http4sLawSuite.scala

```diff
@@ -15,18 +15,20 @@
  */
 
 package org.http4s
-package circe.test // Get out of circe package so we can import custom instances
 
-import cats.effect.IO
-import cats.effect.laws.util.TestContext
-import cats.effect.laws.util.TestInstances._
-import io.circe._
-import io.circe.testing.instances._
-import org.http4s.circe._
-import org.http4s.laws.discipline.EntityCodecTests
+import munit.ScalaCheckEffectSuite
+import org.scalacheck.effect.PropF
 
-// Originally based on ArgonautSuite
-class CirceSpec extends Http4sSpec {
-  implicit val testContext = TestContext()
-  checkAll("EntityCodec[IO, Json]", EntityCodecTests[IO, Json].entityCodec)
+/** Poor's man discipline runner to check a set of PropF[F]
+  */
+trait Http4sLawSuite extends ScalaCheckEffectSuite {
+  def checkAllPropF[F[_]](
```

Comment:
Nit: if `checkAll` is a `Prop`, shouldn't this just be `checkAllF`?

Reply:
Sure, I wasn't really sure how to name it.